packages/flutter/test/cupertino/search_field_test.dart

```diff
@@ -111,7 +111,21 @@ void main() {
       );
     },
   );
-
+  testWidgets('can change keyboard type', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        const CupertinoApp(
+          home: Center(
+            child: CupertinoSearchTextField(
+              keyboardType: TextInputType.text
```

Comment:
This is an analyzer failure because `TextInputType.text` is the default, so this is redundant.

Actually though, wouldn't this test be better if we passed something other than the default?  I think this test would pass on master.  If you change it to `TextInputType.number` and check for that below, it would be more effective.

(I thought I left a comment about this yesterday but I can't find it.  Sorry if this is redundant in case you did receive that).

Reply:
Oh and add a trailing comma here.